src/crypto/password/pbkdf2.clj

```diff
@@ -20,6 +20,9 @@
 (defn- encode-int [i]
   (String. (Base64/encodeInteger (BigInteger/valueOf i))))
 
+(def ^:private default-iterations-count
+  (Long/parseLong (System/getProperty "crypto.password.pbkdf2.default-iterations-count" "100000")))
+
 (defn encrypt
   "Encrypt a password string using the PBKDF2 algorithm. The default number of
   iterations is 100,000. If a salt is not specified, 8 random bytes are
```

Comment:
I left the `8 random bytes` part non-configurable - doesn't seem to fall into the same category of "improve performance"

(and there's a already a `salt` parameter)